Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
 
+
```

Comment:
Den überflüssigen Whitespace hier gerne entfernen. Wir könnten überlegen, hier EditorConfig und Linting einzubauen, um in Zukunft nicht weiter über Formatierungsfragen reden zu müssen.